AdminWebsite/AdminWebsite.UnitTests/Controllers/ConfigSettingsControllerTests.cs

```diff
@@ -45,7 +47,8 @@ public void Should_return_response_with_settings()
                 Options.Create(azureAdConfiguration),
                 Options.Create(kinlyConfiguration),
                 Options.Create(applicationInsightsConfiguration),
-                Options.Create(testSettings)) {
+                Options.Create(testSettings),
+                Options.Create(vhServiceConfiguration)) {
```

Comment:
Is this file missing the check for the VideoWebUrl?

Reply:
We are forced to pass the vhserviceconfiguration from the secrets/appSettings values so itself it checks the url exist. otherwise it will not build at all